src/BadgeImageRendererService.php

```diff
@@ -84,4 +84,17 @@ private function renderBadge(
             Badge::DEFAULT_FORMAT,
         );
     }
+
+    /**
+     * This method required because of native `number_format`
+     * method has big integer format limitation.
+     */
+    private function formatNumber(
```

Comment:
I've removed floats and wrote own string number formatter, because float numbers are formatting incorrectly anyway.